### ISSUE TITLE
update spec to use standard `systemAction`, instead of extension

### ISSUE THREAD

**Issue by isaacvetter** [argonautproject/cds-hooks-for-pama#17]
Following the implementations of this spec, CDS Hooks incorporating a [standard `systemAction` into the CDS Hooks response](https://cds-hooks.org/specification/current/#system-action). This feature will (almost certainly) be published this year in the official HL7 STU2 release of the specification. 

I think it makes sense to update this specification to use the standard systemAction element, instead of using an extension. 

(Feel free to push back and tell me to write a PR myself ;)).

**Comment by mattvarghese** [comment]
Also, any thoughts on how to have guardrails so that the systemAction only updates PAMA info and not other info on the orders? Or is that better asked as a CDS Hooks question?

**Comment by jmandel** [comment]
Agreed @isaacvetter ! Re: updates, you want to share a PR that we can merge when STU2 is published? Or a "work in progress" note to merge sooner?

Re: guardrails, I think they're important and narrative based, but I'd be happy to include narrative in this spec if you want to propose lagnuage @mattvarghese 

(For the broader question of how to provide guardrails and whether they can/should be computable, we should discuss in CDS Hooks)

**Comment by cfeltner** [comment]
Since this specification is already in use on production systems, I think it is important to maintain backwards compatibility. Therefore, I would suggest still allowing the extension but indicate the standard SystemAction be used in future implementations.
